### ISSUE TITLE
[UX] Turn the installer into a 1-minute installer

### ISSUE THREAD

**Issue by Bojhan** [backdrop/backdrop-issues#467]
The Drupal installer takes quite long and frankly, its only because we accumulated a lot of information we need to display. Backdrop can diverge from this and truly provide a 1-2 screen experience. 

For example:
1) Default to English, and provide the option to change it - instead of making it a "step" in the process. 
2) Put Database configuration and admin username/password/e-mail on one page.
3) Skip all the stuff around Site name, site e-mail address (just use admin e-mail), default country, timezone, update notifications. If people want to change that, finding it through the admin UI is much more valuable to build up understanding.

**Comment by jenlampton** [comment]
Updating issue title since Backdrop's installer already takes less than 5 minutes. I don't want to *increase* the amt of time people spend on the installer :)

**Comment by klonos** [comment]
:smile:

**Comment by jenlampton** [comment]
Actually, most installs I do are already less than 1m.  Going to close this issue :)  Please re-open if you object!

**Comment by mikemccaffrey** [comment]
Wow, @jenlampton is really fast at setting up sites! I'm thinking that most users will take more than 1 minute to figure out what everything means.

**Comment by jenlampton** [comment]
Fair enough, we can still tackle the issues outlined in the first post :)

**Comment by klonos** [comment]
> Skip all the stuff around Site name, site e-mail address (just use admin e-mail), default country, timezone, update notifications. If people want to change that, finding it through the admin UI is much more valuable to build up understanding.

We have a chance to do all that now that we have resurrected the Dashboard. We could link to all these things from the new "Welcome to Backdrop!" dashboard block: #3994
